TheMovieDb/HomeViewController.swift

```diff
@@ -7,27 +7,60 @@
 
 import UIKit
 
-class HomeViewController: UIViewController, UICollectionViewDelegate {
+final class HomeViewController: UICollectionViewController {
     var viewMoreButton = UIButton()
-    var collectionView: UICollectionView!
+
     let service = NetworkManager(urlSession: URLSession.shared)
+    static let categoryHeaderUpcoming = "Upcoming"
+    let categoryHeaderUpcomingId = "Upcoming"
     var movies: [Movie] = [] {
         didSet {
             collectionView.reloadData()
         }
     }
+
+    init() {
+        super.init(collectionViewLayout: HomeViewController.createLayout())
+    }
+    
+    static func createLayout() -> UICollectionViewCompositionalLayout {
+        let item = NSCollectionLayoutItem(layoutSize: .init(widthDimension: .fractionalWidth(1), heightDimension: .fractionalHeight(1)))
+        return UICollectionViewCompositionalLayout { (sectionNumber, env) -> NSCollectionLayoutSection in
+            if sectionNumber == 0 {
```

Comment:
No magic numbers, this is risky what if by any reason order change ? wouldn't it be better to use an enum or constant instead ?